package.json

```diff
@@ -11,14 +11,15 @@
     "lint:js": "eslint -c ./tests/linters/.eslintrc.js **/header/header.js **/skip-link/skip-link.js **/feedback-banner/feedback-banner.js",
     "lint:css": "sass-lint -c ./tests/linters/.sass-lint.yml 'packages/**/*.scss' -v -q",
     "lint:html": "htmlhint --config ./tests/linters/.htmlhintrc ./dist/app/components/*.html",
-    "backstop:ref": "backstop reference --config=tests/backstop/backstop.json",
-    "backstop:test": "backstop test --config=tests/backstop/backstop.json",
-    "backstop:clean": "rm -rf tests/backstop/backstop_data/bitmaps_test/*",
+    "backstop:ref": "backstop --config=tests/backstop/backstop.js reference --docker",
+    "backstop:test": "backstop --config=tests/backstop/backstop.js test --docker",
+    "backstop:clean": "rm -rf tests/backstop/bitmaps_test/*",
     "build-gh-pages": "gulp bundle && BASE_URL='/nhsuk-frontend' gulp docs:build",
     "build-gh-release": "gulp zip"
   },
   "dependencies": {
     "accessible-autocomplete": "^1.6.2",
+    "minimist": "^1.2.0",
```

Comment:
@mikemonteith does this need to be a `devDependency`? As it will get installed for people who `npm install nhsuk-frontend`

Reply:
Yep! My bad.